src/Search/Kernel.php

```diff
@@ -514,16 +515,6 @@ public function routes(Application $app)
             ));
     }
 
-    public function handleException(Throwable $e) : Response
-    {
-        $this->app['logger']->error('Failed to serve response', ['exception' => $e]);
```

Comment:
does the ApiProblemProvider set up a log somewhere?

Reply:
Good question, since we're not using the default Monolog provider will need to check.